contracts/VaultOUSD.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import {IERC4626} from "../contracts/interfaces/IERC4626.sol";
+import {ERC4626} from "../contracts/standard/ERC4626.sol";
+
+contract VaultOUSD is ERC4626 {
```

Comment:
I would add a comment around what this Vault is doing (like: only hold OUSD and using 4626 because it deals with re-basing)